compass/ocean/tests/isomip_plus/initial_state.py

```diff
@@ -125,17 +132,21 @@ def _compute_initial_condition(self):
 
         ds['landIceFraction'] = xr.where(mask, ds.landIceFraction, 0.)
 
-        ref_density = constants['SHR_CONST_RHOSW']
-        landIcePressure, landIceDraft = compute_land_ice_pressure_and_draft(
-            ssh=ds.ssh, modify_mask=ds.ssh < 0., ref_density=ref_density)
-
-        ds['landIcePressure'] = landIcePressure
-        ds['landIceDraft'] = landIceDraft
-
-        if self.time_varying_forcing:
-            self._write_time_varying_forcing(ds_init=ds)
+        if thin_film_present:
+            land_ice_thickness = ds.landIceThickness
+            land_ice_pressure = compute_land_ice_pressure_from_thickness(
+                land_ice_thickness=land_ice_thickness, modify_mask=ds.ssh < 0.,
+                land_ice_density=ice_density)
+            land_ice_draft = compute_land_ice_draft_from_pressure(
+                land_ice_pressure=land_ice_pressure,
+                modify_mask=ds.bottomDepth > 0.)
+            ds['ssh'] = np.maximum(land_ice_draft, -ds.bottomDepth)
+        else:
+            land_ice_draft = ds.ssh
+            land_ice_pressure = compute_land_ice_pressure_from_draft(
+                land_ice_draft=land_ice_draft, modify_mask=land_ice_draft < 0.)
 
-        ds['bottomDepth'] = -ds.bottomDepthObserved
+        ds['landIcePressure'] = land_ice_pressure
```

Comment:
```suggestion
        ds['landIcePressure'] = land_ice_pressure
        ds['landIceDraft'] = land_ice_draft
```
This seems to be needed in `_write_time_varying_forcing()` and it is also used in MPAS-Ocean forward mode.

Reply:
@cbegeman, you marked this as resolved but it seems like the code hasn't changed.  Are there changes you have locally that you maybe need to push?

Reply:
Done. Thanks for catching that.